src/zenml/cli/stack_recipes.py

```diff
@@ -1049,10 +1173,13 @@ def deploy(
                             "metadata-db-password' to get the "
                             "value in the command-line."
                         )
+                else:
+                    logger.info("Deployment cancelled.")
+                    return 1
```

Comment:
yup, this case happens when someone says no to the confirmation prompt for checking the values. I think this will go away soon when I start work on refining the logs